CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
 # Contributing Guidelines
-1. Always start by picking a ticket from the backlog. Please chat in Slack if you really want to pick something up and mentors can advise if it seems suitable. If you prefer to be assigned something based on your level that is great too!
+1. Always start by picking a ticket from the backlog [https://trello.com/b/JOTspGgX/sharespotscouk]. Please chat in Slack if you really want to pick something up and mentors can advise if it seems suitable. If you prefer to be assigned something based on your level that is great too!
```

Comment:
Could you use the markdown link syntax here? Like this:

[Text of link](link)

Behind the scenes:
```
[Text of link](link)

```